sdk/lib/rtl/path.c

```diff
@@ -51,6 +52,11 @@ static const UNICODE_STRING RtlpDefaultExtension = RTL_CONSTANT_STRING(L".DLL");
 static const UNICODE_STRING RtlpDotLocal = RTL_CONSTANT_STRING(L".Local\\");
 static const UNICODE_STRING RtlpPathDividers = RTL_CONSTANT_STRING(L"\\/");
 
+/* Search mode used by RtlDosSearchPath_Ustr() and RtlSetSearchPathMode() */
```

Comment:
`RtlDosSearchPath_Ustr` gets a path passed in, it does not calculate it's own path.